src/docs/topics/docker/README.md

```diff
@@ -0,0 +1,203 @@
+
+## Docker compose
+
+Docker Compose will allow you to generate multiple containers locally by doing simply `docker-compose up` command in the root folder of Orchard Core source code. Of course it requires that you have a docker-compose.yml file standing in that folder first. In the example shown below, we will create services for each of the database providers Orchard Core "officially supports".
+
+[Docker Compose documentation](https://docs.docker.com/compose/)
+
+docker-compose.yml file example :  
+
+```YML
+version: '3.3'
+services:
+    web:
+        build: 
+            context: .
+            dockerfile: Dockerfile
+        ports:
+            - "5009:80"
+        depends_on:
+            - sqlserver
+            - mysql
+            - postgresql
+    sqlserver:
+        image: "mcr.microsoft.com/mssql/server"
+        environment:
+            SA_PASSWORD: "P@ssw0rd!123456"
+            ACCEPT_EULA: "Y"
+    mysql:
+        image: mysql:latest
+        restart: always
+        environment:
+            MYSQL_DATABASE: 'orchardcore_database'
+            MYSQL_USER: 'orchardcore_user'
+            MYSQL_PASSWORD: 'orchardcore_password'
+            MYSQL_ROOT_PASSWORD: 'root_password'
+        ports:
+            - '3306:3306'
+        expose:
+            - '3306'
+        volumes:
+            - mysql-data:/var/lib/mysql
+    postgresql:
+        image: postgres:latest
+        volumes:
+            - postgresql-data:/var/lib/postgresql/data
+        ports:
+            - 5432:5432
+        environment:
+            POSTGRES_USER: orchardcore_user
+            POSTGRES_PASSWORD: orchardcore_password
+            POSTGRES_DB: orchardcore_database
+volumes:
+    mysql-data:
+    postgresql-data:
+
+```
+## Prune intermediate images
+
+### When using `docker-compose` command : 
+
+```cmd
+REM Builds images if they are not already built
+docker-compose build
+
+REM Prune intermediate remaining images
+docker image prune -f --filter label=stage=build-env
+
+REM Start all containers
+docker-compose up
+```
+
+We added some commands examples to prune intermediate images because our `Dockerfile` uses an intermediate image to do a `dotnet publish` of our source code. If you don't prune your intermediate images ; over time it can certainly take some significant amount of disk space. We label those intermediate images with `stage=build-env` so that they can be pruned easily.
+
+## Create tenants automatically (Autosetup feature)
+
+TODO
+
+See :
+
+[Database shell configurations](../../reference/core/Shells/README.md#database-shells-configuration-provider)
+
+[Autosetup feature PR](https://github.com/OrchardCMS/OrchardCore/pull/4567)
```

Comment:
Can you remove this until the `Autosetup` feature is actually merged.

Docs shouldn't refer to pr's that may or may not be merged

Reply:
Dudes, I added a TODO on this part because it is. The status of Docker is documented that way we don't have to answer everyone on Gitter about it and I think it's fine. This PR will eventually get merged, why wouldn't it be?

Reply:
Or maybe I should be clearer by adding the TODO on the PR link. This part in incomplete anyways.

Reply:
Nah I don't agree with you guys at all on that one. Should I not use Buildkit too then because it doesn't work completely? I mean I documented the whole Docker story even things that works and things that don't and that's how a documentation should be.